ini.go

```diff
@@ -63,8 +63,10 @@ func iniSave(filename string, iniFile *ini.File) error {
 	if err := os.Chmod(f.File.Name(), mode); err != nil {
 		return err
 	}
-	if err := os.Chown(f.File.Name(), uid, gid); err != nil {
-		return err
+	if os.Geteuid() == 0 {
```

Comment:
I am not convinced by this change. It would mean that:

* If you run as root, ownership will be respected
* If you run as non-root, ownership may not be respected, and you will not know about it at first

I would instead suggest something such as:

```suggestion
	if os.Geteuid() != uid {
```

This would only try to change the ownership when needed. If you were running as a non-root container and changing a ini file with a different ownership, it would fail anyways during the "Commit" with a permission denied error so I don't think it should be a problem.

Reply:
Done but also checkin gid